### PR TITLE
Convert ActionBar to functional component

### PR DESCRIPTION
**Pull request by flacoman91**
Convert ActionBar to functional component

## Additions

- Better tests

## Removals

- Snapshot test

## Changes

-

## Testing

- run tests, everything should still work

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/components/ActionBar/ActionBar.spec.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    expect(buttonExport).toBeInTheDocument();
+    fireEvent.click(buttonExport);
+    expect(dataExportSpy).toHaveBeenCalledTimes(1);
+
+    const buttonPrint = screen.getByRole('button', { name: 'Print' });
+    expect(buttonPrint).toBeInTheDocument();
+    fireEvent.click(buttonPrint);
+
+    expect(window.location.assign).toHaveBeenCalledWith(
+      'http://ccdb-website.gov&isPrintMode=true&' + 'isFromExternal=true'
```
Comment (KatherynRizek) [inline src/components/ActionBar/ActionBar.spec.js:73]: Why is this broken up into two different strings?

Comment (KatherynRizek) [reply src/components/ActionBar/ActionBar.spec.js:73]: If the order of the the url isn't guaranteed, you could use
```
expect(window.location.assign).toHaveBeenCalledWith(
      expect.stringContaining('http://ccdb-website.gov&isPrintMode=true&') &&
        expect.stringContaining('isFromExternal=true')
    );
```

Comment (flacoman91) [reply src/components/ActionBar/ActionBar.spec.js:73]: i missed some things when i copy pasted.  i think the linter probably changed and didn't fix my formatting.

Comment (flacoman91) [reply src/components/ActionBar/ActionBar.spec.js]: updated.  The order shouldn't change.  The original code also had this linting warning.

**Review by KatherynRizek — APPROVED** [review]

**Pull request merged**